modules/phase_field/include/postprocessors/NodalFloodCount.h

```diff
@@ -58,20 +58,25 @@ class NodalFloodCount :
   virtual Real getNodalValue(dof_id_type node_id, unsigned int var_idx=0, bool show_var_coloring=false) const;
   virtual Real getElementalValue(dof_id_type element_id) const;
 
-  virtual const std::vector<std::pair<unsigned int, unsigned int> > & getNodalValues(dof_id_type /*node_id*/) const;
-  virtual std::vector<std::vector<std::pair<unsigned int, unsigned int> > > getElementalValues(dof_id_type /*elem_id*/) const;
+  virtual Real getEntityValue(dof_id_type entity_id, unsigned int var_idx=0, bool show_var_coloring=false) const;
+
+//  virtual const std::vector<std::pair<unsigned int, unsigned int> > & getNodalValues(dof_id_type /*node_id*/) const;
```

Comment:
Yes, all of this NodalValues() code is currently commented, there's more below. Haven't decided if it'll go permanently or stay. I'd like to see it go, but several people have used it. It may go in the near future in which case I'll follow this up with another commit to do more cleanup and refactoring.